client/src/components/ScheduledAppointments/YourAppointments.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Typography, makeStyles } from "@material-ui/core";
+
+const YourAppointments = ({ upcoming, empty }) => {
+  const classes = useStyles();
+  if (empty) {
+    return (
+      <Typography className={classes.text} variant="body2">
+        You do not have any {upcoming ? "upcoming" : "past"} appointments
+      </Typography>
+    );
+  } else {
+    return (
+      <Typography className={classes.text} variant="h4">
+        Your {upcoming ? "upcoming" : "past"} appointments
+      </Typography>
+    );
+  }
+};
```

Comment:
As mentioned in comment on PR, maybe leave the heading always there, and just make the "you do not have any..." conditional